frontend/prebuild/package.json

```diff
@@ -13,48 +13,49 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^5.2.10",
-    "@angular/cli": "^1.7.4",
-    "@angular/common": "^5.2.10",
-    "@angular/compiler": "^5.2.10",
-    "@angular/core": "^5.2.10",
-    "@angular/forms": "^5.2.10",
-    "@angular/http": "^5.2.10",
-    "@angular/platform-browser": "^5.2.10",
-    "@angular/platform-browser-dynamic": "^5.2.10",
-    "@angular/router": "^5.2.10",
-    "@ng-bootstrap/ng-bootstrap": "^1.1.2",
-    "@types/createjs": "0.0.29",
-    "angular-in-memory-web-api": "^0.5.4",
+    "@angular/animations": "^6.0.2",
+    "@angular/common": "^6.0.2",
+    "@angular/compiler": "^6.0.2",
+    "@angular/core": "^6.0.2",
+    "@angular/forms": "^6.0.2",
+    "@angular/http": "^6.0.2",
+    "@angular/platform-browser": "^6.0.2",
+    "@angular/platform-browser-dynamic": "^6.0.2",
+    "@angular/router": "^6.0.2",
+    "@ng-bootstrap/ng-bootstrap": "^2.0.0",
+    "angular-in-memory-web-api": "^0.6.0",
     "autoprefixer": "^7.2.6",
     "bootstrap": "^4.1.0",
-    "core-js": "^2.5.5",
-    "createjs": "^1.0.1",
+    "core-js": "^2.5.6",
     "createjs-module": "^0.8.3",
-    "node-sass": "^4.8.3",
-    "npm": "^5.8.0",
-    "rxjs": "^5.5.10",
+    "ng-event-source": "^1.0.10",
+    "node-sass": "^4.9.0",
+    "npm": "^5.10.0",
+    "rxjs": "^6.0.0",
+    "rxjs-compat": "^6.1.0",
     "web-animations-js": "^2.3.1",
     "zone.js": "^0.8.26"
   },
   "devDependencies": {
-    "@angular/compiler-cli": "^5.2.10",
-    "@angular/language-service": "^5.2.10",
-    "@types/jasmine": "~2.8.3",
+    "@angular-devkit/build-angular": "^0.6.1",
+    "@angular/cli": "^6.0.1",
+    "@angular/compiler-cli": "^6.0.2",
+    "@angular/language-service": "^6.0.2",
+    "@types/jasmine": "^2.8.7",
     "@types/jasminewd2": "~2.0.2",
-    "@types/node": "^6.0.106",
-    "codelyzer": "^4.2.1",
+    "@types/node": "^6.0.110",
+    "codelyzer": "^4.3.0",
     "jasmine-core": "~2.8.0",
     "jasmine-spec-reporter": "~4.2.1",
-    "karma": "~2.0.0",
+    "karma": "^2.0.2",
     "karma-chrome-launcher": "~2.2.0",
     "karma-cli": "~1.0.1",
     "karma-coverage-istanbul-reporter": "^1.4.2",
-    "karma-jasmine": "~1.1.0",
+    "karma-jasmine": "^1.1.2",
     "karma-jasmine-html-reporter": "^0.2.2",
     "protractor": "~5.1.2",
     "ts-node": "~3.2.0",
     "tslint": "~5.9.1",
-    "typescript": "~2.5.3"
+    "typescript": ">=2.7.2 <2.8.0"
```

Comment:
why downgrading typescript?

Reply:
How do you mean? 2.7 is newer...

Reply:
oops, managed to read the diff backwards...